src/main/java/ShortSet.java

```diff
@@ -1,3 +1,5 @@
+import java.util.*;
+
 public class ShortSet {
   public static void main (String[] args) {
```

Comment:
Good job! The static analysis tool Error Prone reported that the below error [1] used to be here, but you removed it. Check out http://errorprone.info for more information.


[1] ```RecommenderTest/src/main/java/ShortSet.java:4: error: cannot find symbol
    Set<Short> s = new HashSet<>();
    ^
  symbol:   class Set
  location: class ShortSet```